### PR TITLE
I've added Derpy Murky, Master League and both Valeera flair.

### PR DESCRIPTION
**Pull request by kalebhermes**
I put Valeera flair at the bottom of the current spritesheet for flair and updated hero_flair.scss.

I put Derpy Murky and Master League into the new flair-general, and updated general.scss with the new position for scilenced.

### CODE REVIEW

File: user_flair/general.scss [hunk 1]
```diff
@@ -3,5 +3,13 @@
...
 .flair-g-silenced::before {
-  background-position: 0 0;
+  background-position: 40px -260px;
 }
+
+.flair-g-derpy-murky::before {
+	background-position: 40px -280px;
+}
+
+.flair-g-master-league::before {
```
Comment (yuna9) [inline user_flair/general.scss]: Could we make this `masterleague` instead of `master-league`?

**Review by yuna9 — CHANGES_REQUESTED** [review]
Looks good, just one small comment

File: user_flair/general.scss (hunk 1 above)
Comment (kalebhermes) [reply user_flair/general.scss]: That seems like a fine idea. Will change now.

File: user_flair/general_flair.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.flair-general::before {
```
Comment (yuna9) [inline user_flair/general_flair.scss:1]: If renaming the file, the import in `main.scss` should also be updated.

Comment (kalebhermes) [reply user_flair/general_flair.scss:1]: Damnit.

**Review by yuna9 — APPROVED** [review]
:package: :ship: :moneybag:

**Pull request merged**